### DISCUSSION TITLE
Organization Profile will not shown

### DISCUSSION THREAD

**Discussion by sjuergen** [simatic-ax/.discussions#5] [Q&A] [answered]
On the start page of SIMATIC AX is profile shown where a image is missing.
https://github.com/simatic-ax

![image](https://user-images.githubusercontent.com/49430012/175323233-74a7e136-02e6-429c-b04e-e4c2630f5f23.png)

But in the related README.md its correct
https://github.com/simatic-ax/.github/tree/main/profile

Any idea, what's wrong?

**Comment by RauerJ** [comment, marked as answer]
The Image Link points to
https://github.com/simatic-ax/.github/blob/main/images/subscribe_notifications.png
but the image is located at
https://github.com/simatic-ax/.github/blob/main/profile/images/subscribe_notifications.png

I believe it does not work since you are using a relative path in the readme in .github (which is actually in .github/profile) and the "/profile" part is lost on the main page.

**Reply by sjuergen** [reply]
Good hint. So I stored the image under the other path
